src/assets/index.js

```diff
@@ -0,0 +1,3 @@
+import FooterLogo from './footer-logo.png';
+
+export default FooterLogo;
```

Comment:
Do not export default in index.js, it's supposed to group all the exports from the folder and there can only be one default export